packages/zowe-explorer/src/configuration/Constants.ts

```diff
@@ -0,0 +1,344 @@
+            value: `Active`,
+            picked: false,
+        },
+        {
+            key: `Input`,
+            label: `Input`,
+            value: null,
+            picked: false,
+        },
+        {
+            key: `Output`,
+            label: `Output`,
+            value: null,
+            picked: false,
+        },
+    ];
+
+    export const JOB_STATUS_UNSUPPORTED = [
+        {
+            key: `All`,
+            label: `*`,
+            value: null,
+            picked: true,
+        },
+    ];
+
+    export enum JobPickerTypes {
+        QuerySearch = "QuerySearch",
+        IdSearch = "IdSearch",
+        History = "History",
+    }
+
+    export const SEPARATORS = {
+        BLANK: { kind: vscode.QuickPickItemKind.Separator, label: "" },
+        RECENT: { kind: vscode.QuickPickItemKind.Separator, label: vscode.l10n.t("zowe.separator.recent", "Recent") },
+        RECENT_FILTERS: { kind: vscode.QuickPickItemKind.Separator, label: vscode.l10n.t(`Recent Filters`) },
+        OPTIONS: { kind: vscode.QuickPickItemKind.Separator, label: vscode.l10n.t(`Options`) },
+    };
+
+    /**
+     * Defines all global variables
+     * @param tempPath File path for temporary folder defined in preferences
+     */
+    export function defineGlobals(tempPath: string | undefined): void {
+        SETTINGS_TEMP_FOLDER_LOCATION = tempPath;
+        // Set temp path & folder paths
+        ZOWETEMPFOLDER = tempPath ? path.join(tempPath, "temp") : path.join(__dirname, "..", "..", "resources", "temp");
+        ZoweLogger.info(
+            vscode.l10n.t({
+                message: `Zowe Explorer's temp folder is located at {0}`,
+                args: [ZOWETEMPFOLDER],
+                comment: ["Zowe temp folder"],
+            })
+        );
+        ZOWE_TMP_FOLDER = path.join(ZOWETEMPFOLDER, "tmp");
+        USS_DIR = path.join(ZOWETEMPFOLDER, "_U_");
+        DS_DIR = path.join(ZOWETEMPFOLDER, "_D_");
+    }
+
+    export function setConfigPath(configPath: string | undefined): void {
+        if (configPath) {
+            CONFIG_PATH = configPath;
+        } else {
+            CONFIG_PATH = FileManagement.getZoweDir();
+        }
+    }
+
+    export function setActivated(value: boolean): void {
+        if (value) {
+            ZoweLogger.info(vscode.l10n.t(`Zowe Explorer has activated successfully.`));
+        }
+        ACTIVATED = value;
+    }
+
+    export function setSavedProfileContents(value: Uint8Array): void {
+        SAVED_PROFILE_CONTENTS = value;
+    }
+
+    export function setCompareSelection(val: boolean): void {
+        FILE_SELECTED_TO_COMPARE = val;
+        vscode.commands.executeCommand("setContext", "zowe.compareFileStarted", val);
+    }
+
+    export function resetCompareChoices(): void {
+        setCompareSelection(false);
+        filesToCompare = [];
+    }
+
+    export function setProfilesCache(profilesCache: Profiles): void {
+        PROFILES_CACHE = profilesCache;
+    }
```

Comment:
Curious if some of these functions can be removed? I think they were needed when variables like `CONFIG_PATH` and `PROFILES_CACHE` were top-level exports. Now since they've been moved inside the `Constants` class, we should be able to assign `Constants.PROFILE_CACHE = x` from elsewhere.